### ISSUE TITLE
No way to tell what the darknet binary prints.

### ISSUE THREAD

**Issue by Huxwell** [AlexeyAB/darknet#7394]
When I train yolo v4, I get a lot of outputs that I later use to plot metrics.
With max_batches = 1543 darknet prints 74064 outputs (with loss etc)
With batch = 32, subdivisions = 16, I have only 49376 itarations (defined as single images passed through network)!

Something is definitely off, does anybody know how often this binary is supposed to return values?
Second question: is there a flag that would allow to print current learning rate? I've seen it being returned in different use cases of darknet.

```
./darknet detector train data/obj.data ../yolov4-custom.cfg yolov4.conv.137 -dont_show -map &> logs.txt & disown
```
```
v3 (iou loss, Normalizer: (iou: 0.07, cls: 1.00) Region 139 Avg (IOU: 0.000000, GIOU: 0.000000), Class: 1.000000, Obj: 0.000000, No Obj: 0.000000, .5R: 0.000000, .75R: 0.000000, count: 1, class_loss = 0.500000, iou_loss = 0.000000, total_loss = 0.500000 
v3 (iou loss, Normalizer: (iou: 0.07, cls: 1.00) Region 150 Avg (IOU: 0.394489, GIOU: 0.289983), Class: 0.985003, Obj: 0.004027, No Obj: 0.000896, .5R: 0.000000, .75R: 0.000000, count: 6, class_loss = 3.253636, iou_loss = 0.891068, total_loss = 4.144704 
v3 (iou loss, Normalizer: (iou: 0.07, cls: 1.00) Region 161 Avg (IOU: 0.612244, GIOU: 0.555591), Class: 0.897004, Obj: 0.014743, No Obj: 0.000339, .5R: 1.000000, .75R: 0.000000, count: 1, class_loss = 0.492179, iou_loss = 0.095293, total_loss = 0.587472 
```

I

**Comment by LeRoii** [comment]
max_batches=1543  means the whole training process has 1543 batches 
batch=32 means there are 32 images in every batch
subdivisions = 16 means every batch is divided into 16 subset, in every subset there are 2 images
1543*32=49376 is the number of image fed to the model for training

when a batch is finished training, there will be an output like this
```
(next mAP calculation at 16322 iterations)                                                                                                               
Last accuracy mAP@0.5 = 96.08 %, best = 96.27 %                                                                                                          
15999: 40.254128, 47.023991 avg loss, 0.000010 rate, 3.221427 seconds, 1023936 images, 0.133928 hours left
```
when a subset is finished training, there will be an output like this
```
v3 (iou loss, Normalizer: (iou: 0.07, obj: 1.00, cls: 1.00) Region 139 Avg (IOU: 0.758198), count: 3, total_loss = 16.571793                              
v3 (iou loss, Normalizer: (iou: 0.07, obj: 1.00, cls: 1.00) Region 150 Avg (IOU: 0.759290), count: 6, total_loss = 11.304995                              
v3 (iou loss, Normalizer: (iou: 0.07, obj: 1.00, cls: 1.00) Region 161 Avg (IOU: 0.893880), count: 9, total_loss = 2.076211                               
total_bbox = 24537007, rewritten_bbox = 0.830374 % 
```
